File: app/src/main/java/com/example/localizationjava/MainActivity.java
package com.example.localizationjava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button btn_press;
    Button btn_shared_prefs;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();
    }

    private void initViews() {

        btn_press = findViewById(R.id.btn_press);
        btn_shared_prefs = findViewById(R.id.btn_shared_prefs);

        btn_press.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), LanguageActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/localizationkotlin/MainActivity.kt
package com.example.localizationkotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    lateinit var btn_press: Button
    lateinit var btn_shared_prefs: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initViews()
    }

    private fun initViews() {
        btn_press = findViewById(R.id.btn_press)
        btn_shared_prefs = findViewById(R.id.btn_shared_prefs)

        btn_press.setOnClickListener{
            val intent = Intent(applicationContext, LanguageActivity::class.java)
            startActivity(intent)
        }

        btn_shared_prefs.setOnClickListener{
            val intent = Intent(applicationContext, PreferenceActivity::class.java)
            startActivity(intent)
        }
    }
}